docs-site/docs/guides/transactions.md

```diff
@@ -0,0 +1,178 @@
+
+### Automatic vs Manual tracking of transactions
+
+<!-- For example, via the [ethers library](https://docs.ethers.org/v5/api/utils/transactions/) or whichever library you're using. -->
+
+If you are automatically tracking transactions with the non-react `.init()` method, the SDK automatically tracks the `TRANSACTION_TRIGGERED` event and `TRANSACTION_SUBMITTED` events for you, but **only for MetaMask**.
+
+If you are using the React Provider, **you are required need to manually track transactions**.
+
+Event with the non-react `.init()` method, manual tracking allows for more fine-grained control over the tracking process.
+
+---
+
+## How to manually track the transaction process
+
+### Required events
+
+When manually tracking the transaction process, **you are required to at a minimum track the following events with the SDK**, so 0xArc has the necessary information to match your SDK events to the blockchain:
+
+1. [Wallet Connection](/tracking/wallet)
+2. [Transaction](/tracking/transaction) - Required for us to have the `transactionHash` to match the transaction against the blockchain
+
+**Note that if you do not track these 2 events, the SDK will not be able to match the transaction to the blockchain, and the transaction will not be shown in the 0xArc App.**
+
+Although not strictly required, we strongly encourage you to also track the following events:
+
+3. [Chain Changed](/tracking/chain) - Required for us to know which chain the transaction took place on
+4. [Signing Events](/tracking/signature) - for a complete picture
+
+### React Example of Required Events
+
+The below is an example which shows how you would track the minimum required events to track a transaction, with the `@web3-react/core` library.
+
+```tsx
+import { useArcxAnalytics } from '@0xarc-io/analytics'
+import { useWeb3React } from '@web3-react/core'
+
+const Component = () => {
+  const sdk = useArcxAnalytics()
+  const { account, chainId } = useWeb3React()
+
+  // 1. This is where you emit the wallet connection once the user has connected their web3 wallet
+  useEffect(() => {
+    if (account && chainId) {
+      sdk.wallet({ account, chainId })
+    }
+  }, [account, chainId, sdk])
+
+  // 2. The user has triggered a transaction. We send this to the SDK with the transaction hash
+  const sendTransaction = async () => {
+    // Example: Simulating a transaction call
+    // In a real scenario, you would replace this with your transaction logic,
+    // for example, using ethers.js or web3.js to interact with a smart contract
+
+    const transactionHash = '0x023c0e7...' // Placeholder for the actual transaction hash
+
+    // Assuming the transaction was successful and you have the hash
+    // Now, track the transaction using the analytics SDK
+    sdk.transaction({
+      transactionHash,
+      account,
+      chainId,
+    })
+  }
+
+  return (
+    <>
+      <button>Open Wallet</button>
```

Comment:
to remove? this button has no connection with the example (no callback is associated with it)